orchestra/management/commands/loadworkflow.py

```diff
@@ -0,0 +1,35 @@
+from django.core.management.base import BaseCommand, CommandError
+from orchestra.workflow.exceptions import WorkflowError
+from orchestra.workflow.load import load_workflow
+
+import os
+
+
+class Command(BaseCommand):
+    help = 'Loads a workflow version into the database'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'workflow_directory',
+            help='The code directory containing the workflow.')
+        parser.add_argument(
```

Comment:
Currently the semantics are "if it is provided, load the version. If not, load all versions." Are you saying it should be "a single version must be provided and that version will be loaded"?